CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - Fix bug where `dtype` was not used when creating a `ParameterBeam` from Twiss parameters (see #206) (@jank324)
 - Fix bug after running `Segment.inactive_elements_as_drifts` the drifts could have the wrong `dtype` (see #206) (@jank324)
 - Fix an issue where splitting elements would result in splits with a different `dtype` (see #211) (@jank324)
+- Fix an issue where the `dtype` and `device` are set back to default when broadcasting elements or beams. (see #218) (@cr-xu)
```

Comment:
I think #218 should just be listed in the PRs for the vectorisation change because this changelog is with respect to `v0.6.3` where this bug actually didn't exist.